### PR TITLE
Revenant Balance Check

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

Reverts the deadchat part of https://github.com/yogstation13/Yogstation/pull/10735 because you are IN the round, you SHOULD NOT be able to see or hear deadchat. Wizard scrying orb was removed for a similar reason (yes different but very similar). this was the ORIGNAL INTENT FIVE FUCKING YEARS AGO.
revenants being spoon fed OOC info as an IN-CHARACTER ANTAG, is fucking abysmal. Being told right where a body is with no need to go like...scout for it, is unbalanced to the core.

Makes Revenants unable to fire abilities on space turfs, because camping outside the evac area in space spamming abilities is cringe as hell

makes Defile unable to damage Reinforced, Plasma, Reinforced Plasma, and Plastitanium windows, because also spacing an area or loosing a plasma or fusion flood with one or two ability casts is NOT ok. Firelocks dont even help because it will still break THOSE too leaving you with a revenant capable of spacing important areas.

# Changelog

:cl:  
rscdel: Revenants cannot hear dead chat anymore, as participants IN a round
tweak: Defile cannot damage non-standard windows (reinforced/plasma/etc)
tweak: Revenants cannot use abilities in space
/:cl:

### CODE REVIEW

**Comment by LazennG** [comment]
don't think this is good cause
1) rev is for getting the spite of ghosts to be acted on in a round
2)they can be killed real fast so space is needed so they can actually cause havoc around people and not ssd bodies in dorms
3)most windowed areas have those emergency doors so the areas aren't spaced after the windows get smashed

**Comment by VaelophisNyx** [comment]
counter point, a rev can just keep spamming defile until the firelocks break, and spaces the area, or floods it with plasma/fusion/n2o/whatever

Revenant can still go cause chaos and inact revenge, it just cant be a salty little shit over it and camp in space

**Comment by isotope436** [comment]
counter to your counter learn how to use a salt nade and then make it so revs cant kill shuttle windows and you solve all your issues as EVERY WINDOW ON STATION IS REINFORCED

**Comment by isotope436** [comment]
the funny space tile thing i can understand but just make it so that it cant appear in space and your golden

**Comment by VaelophisNyx** [comment]
> counter to your counter learn how to use a salt nade and then make it so revs cant kill shuttle windows and you solve all your issues as EVERY WINDOW ON STATION IS REINFORCED

the problem wasnt even shuttle windows

the engine windows there are just reinforced glass
the windows at evac, are just reinforced glass
the windows atmosians use to hold in fusion reactions are....reinforced....glass.

its an issue.

**Comment by Mqiib** [comment]
You realized if a SINGLE person had thought to grab a laser during the five minutes I spent on those windows the round before you made this rage pr I would have been absolutely powerless against it and would have had to stop, right? Just because nobody bothered to grab something to actually DEAL with me, the revenant, beyond salt and laser scalpels doesn't mean that revenants are OP in some way. 
As for the glass thing, revenants can't do anything to atmos holofans or solid walls so if you really wanted to stop me maybe someone should have put down more of those, or perhaps the chemist spamming advanced release repulse grenades might have been part of the issue with people trying to get onto the shuttle and being thrown around all the time. Plus, if there's an atmos tech doing open tile fusion and only separating it with glass windows and not something a little more robust like full tile plasma glass or a solid wall, that's poor planning on their part. The only change I think would be acceptable is making specifically plasma glass immune to revenants because it's already effectively impossible to destroy for them in any reasonable amount of time.
For ghost chat, it's not an issue. Yes ghosts can tell them things but revenants have very few actual tools to directly impact the round beyond malfunction without putting themselves at serious risk of being destroyed. It's also a lot of fun being able to talk with ghosts and just hang out if there isn't anything to do.

tldr: This is a kneejerk reaction to a particularly nasty revenant round that just played into the hand of the revenant who was exceptionally aggressive because there were no other active antags. The nerfs aren't warranted, you're just mad.

**Comment by VaelophisNyx** [comment]
the nerfs are warranted, you're just mad that I'm doing something about an antag that needs its balance brought back in line.

**Comment by Marmio64** [comment]
Talking to ghosts and trying to work with them is an essential part of rev gameplay and one of it's distinguishing characteristics imo.

**Comment by VaelophisNyx** [comment]
it was not intended in the design of the antagonist, and was tacked on as part of a fix for them being deaf
the fact that it got through is silly

**Comment by Marmio64** [comment]
> it was not intended in the design of the antagonist, and was tacked on as part of a fix for them being deaf the fact that it got through is silly

Doesn't matter if it was part of the original design, it can still become essential. SS13 was made as a atmospherics sim yet now antags are an essential part of pretty much every server.

**Comment by VaelophisNyx** [comment]
this has no relevance to an antagonist being spoon fed OOC info.

**Comment by kooarbiter** [comment]
revs are milktoast and underwhelming IMO, giving them deadchat access doesn't really enable them to do much more than find an ssd body in maint they might not have the patience to go looking for.

Not casting abilities in space is fine, but maybe let them destroy windows, just have it take a long time? If nobody is watching departures for a solid 5 minutes or however long and the rev breaches a window, then that's the crews fault, and an easy fix tbh

**Comment by VaelophisNyx** [comment]
except I have EXPERIENCED deadchat being an incredibly powerful tool for revenants to just skip all gameplay in finding their essence sources.

**Comment by Skrem7** [comment]
> You realized if a SINGLE person had thought to grab a laser during the five minutes I spent on those windows the round before you made this rage pr I would have been absolutely powerless against it and would have had to stop, right? Just because nobody bothered to grab something to actually DEAL with me, the revenant, beyond salt and laser scalpels doesn't mean that revenants are OP in some way. As for the glass thing, revenants can't do anything to atmos holofans or solid walls so if you really wanted to stop me maybe someone should have put down more of those, or perhaps the chemist spamming advanced release repulse grenades might have been part of the issue with people trying to get onto the shuttle and being thrown around all the time. Plus, if there's an atmos tech doing open tile fusion and only separating it with glass windows and not something a little more robust like full tile plasma glass or a solid wall, that's poor planning on their part. The only change I think would be acceptable is making specifically plasma glass immune to revenants because it's already effectively impossible to destroy for them in any reasonable amount of time. For ghost chat, it's not an issue. Yes ghosts can tell them things but revenants have very few actual tools to directly impact the round beyond malfunction without putting themselves at serious risk of being destroyed. It's also a lot of fun being able to talk with ghosts and just hang out if there isn't anything to do.
> 
> tldr: This is a kneejerk reaction to a particularly nasty revenant round that just played into the hand of the revenant who was exceptionally aggressive because there were no other active antags. The nerfs aren't warranted, you're just mad.

Ah yes assuming someone can just grab a laser willy-nilly to deal with the revenant, which assumes not only that the armory can be opened but that the people who can open it are smart enough to do so. Also that they have enough time to actually traverse the station since it sounds like you dropped a heat bomb through every single corridor (very cool).

Also revenants can EMP (bye-bye laser gun), or hide behind a wall next to a window or behind a console because lasers can't go over those. It's not hard to position in such a way where you can't be shot in space. The walls are reflective and it's more likely that people will shoot others in the shuttle than you.

Also also assuming people just have atmos holofans, the thing strictly limited to atmos lockers or the engi protolathe (then also locked behind technology and plastic). Engineers not throwing down walls is a player issue, though.

The fusion point is entirely valid. Shitty containment of fusion that doesn't involve plasmaglass or rwalls is absurd and should be discouraged. Heat can grow high enough that rglass is just straight broken. They should know better.

Regardless, your attitude is largely dismissive of the larger problems at hand here and an antag that can go and just release fusion tends to not really be fun for all the people who just suddenly husk with little to no time to react. It's one reason why people hate malf AI. Enabling another antag to do it that's a particularly slippery midround that doesn't really add much of an interesting dynamic to the shift is downright comedic. I don't think revenants are OP but they're just not fun in my eyes. Simple as.

**Comment by VaelophisNyx** [comment]
> it sounds like you dropped a heat bomb through every single corridor (very cool).

Not Mqiib in this case, they just spaced Evac unopposed
but I have witnessed a revenant loose plasma on multiple occasions as well as have seen screenshots of a round where one loosed a fusion-flood. The issue with the latter being that directional windows only break if they're on the same time as the hot-spot so atmosians WILL abuse that and put normal ass glass on the adjacent tile facing towards their intended burn chamber, so its untouched by the heat

**Review by JamieD1 — APPROVED** [review]
I do agree revenant in the hands of someone competent can be a real pain in the ass, knowing mqiib he played it annoying well to the point you made this PR. I want to see how badly this nerfs them because I do agree, they can completely take over a round if they know what they are doing.

**Comment by VaelophisNyx** [comment]
the funny thing is revenants seem to be operating better with this merged, lol
